packages/types/src/documents/global/config.ts

```diff
@@ -108,4 +108,5 @@ export enum ConfigType {
   GOOGLE = "google",
   OIDC = "oidc",
   OIDC_LOGOS = "logos_oidc",
+  SCIM = "scim",
```

Comment:
Could you put together a type for this? Just to model the expected fields i.e. `scimEnabled`